program/find-average-of-numbers-by-recursion/FindAverageOfNumbersByRecursion.ts

```diff
@@ -0,0 +1,12 @@
+function average_of_numbers(arr: any, length: number, i: number): any {
+    if (i == length - 1) {
+      return arr[i];
+    }
+    if (i == 0) return (arr[i] + average_of_numbers(arr, length, i + 1)) / length;
+  
+    return arr[i] + average_of_numbers(arr, length, i + 1);
```

Comment:
can we use if{  } else if {  } else { } for a better readability ?

Reply:
review comments incorporated! Thanks